apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -318,19 +318,27 @@ class TopInstructionsCSP extends Component {
       (this.props.referenceLinks && this.props.referenceLinks.length > 0);
 
     const displayHelpTab = videosAvailable || levelResourcesAvailable;
-    const displayFeedbackStudent =
+
+    const studentHasFeedback =
       this.props.viewAs === ViewType.Student &&
       this.state.feedbacks.length > 0 &&
       (this.state.feedbacks[0].comment || this.state.feedbacks[0].performance);
-    const displayFeedbackTeacher =
-      this.state.displayFeedbackTeacherFacing ||
-      (!this.state.displayFeedbackTeacherFacing &&
-        this.props.viewAs === ViewType.Teacher &&
-        this.state.rubric);
-    const displayFeedback = displayFeedbackStudent || displayFeedbackTeacher;
+
+    const displayKeyConcept =
+      this.state.rubric &&
+      ((this.props.viewAs === ViewType.Student && !studentHasFeedback) ||
+        (this.props.viewAs === ViewType.Teacher &&
```

Comment:
Could you please add a comment on the definition of displayKeyConcept here? Something like "if there is no feedback to display, ...". I think the logic is pretty well structured, I think I may have just lost track of what it means to display a key concept here.